### ISSUE TITLE
Inter-mod compatibility issue (1.16.5)

### ISSUE THREAD

**Issue by Xiaolin2-g** [FoxMcloud5655/Draconic-Additions#53]
For some reason, this mod prevents some dimensions from being able to be accessed, like JAMD (Just Another Mining Dimension), Twilight Forest, and The Undergarden.

**Comment by FoxMcloud5655** [comment]
Gonna need more details than that.  What exactly happens?  You go through a portal and it crashes?  It hangs indefinitely?  They simply don't work?  A message pops up in chat?  What version of Draconic Additions are you specifically using?  What version of Forge?  Which mod can reliably be used to test this incompatibility?  Which version of the other mod does this incompatibility happen on?

**Comment by Xiaolin2-g** [comment]
1- The portals simply don't work, the ones that you have to traverse like Twilight Forest one does nothing, and if you have to right click it (JAMD), it says "Invalid world ID set".
In the save folder, there is a "dimensions" folder. Well, only some dimensions are being created there, like Advent of Ascension's ones. The three mods that I mentioned don't create anything, there could be even more mods with this problem.

2- **Draconic-Additions-1.16.5-2.2.2.5-universal** (July 10, 2022).

3- **Forge 36.2.35**

4- When removing mods in batches to search for the culprit, I was testing it with JAMD dimension.
Version **JAMD-1.2.4-build.17+mc1.16.5.jar** (Nov 27, 2021).

**Comment by FoxMcloud5655** [comment]
Thank you. That's enough information for me to go test on my own.

**Comment by UnRealDinnerbone** [comment]
JAMD mod author here, my guess is something is wrong with your JSON files, causing any mod to load after your to not load there. the "Invalid world ID set" is my message for when it cant find the world

**Comment by FoxMcloud5655** [comment]
I don't mess with dimension files at all, nor do I add any.  I used to before the rework of an item, but all references to them have been purged.  Maybe something is left over from that, but I don't deal with DIMs.

**Comment by UnRealDinnerbone** [comment]
I see files about them [here](https://github.com/FoxMcloud5655/Draconic-Additions/tree/1.16.5/src/main/resources/data/draconicadditions). even if you don't reference them anywhere they still load

**Comment by FoxMcloud5655** [comment]
Just getting back to this.  I had my data generator set up to copy the files over when it ran to be included with the final build.  I removed all references to the files, and despite them not being in the generated folders, they still loaded in.  I guess anything in the data folder of the main assets are already copied over by default?  Because when I tried leaving them in there before, it didn't work.

Either way, this is going to be removed completely in the next build and should fix this issue once and for all.
